### DISCUSSION TITLE
Cannot successfully interview ZEN 20 device

### DISCUSSION THREAD

**Discussion by Sraw** [zwave-js/zwave-js#7481] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?

When trying to add ZEN 20 device to ZWave JS, the interview progress is stuck at the NodeInfo status. And also report the device as dead although I can ping the device.

What did you expect to happen instead?

Add the device without issue.

  1. Just add ZEN 20 Device using the LR protocol.

### Software versions

zwave-js-ui: 9.27.8.6c82b1d

zwave-js: 14.3.7

### Z-Wave Controller (Stick/Dongle/...)

discussions

### Device information

Manufacturer: Zooz
Model name: Zen 20
Node ID: 271

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_current.log](https://github.com/user-attachments/files/18083637/zwavejs_current.log)

**Comment by AlCalzone** [comment]
Preliminary investigation: It looks like the device crashes after this command:
```
2024-12-10T17:19:39.823Z DRIVER » [Node 271] [REQ] [SendDataBridge]
                                  │ source node id:   1
                                  │ transmit options: 0x25
                                  │ callback id:      150
                                  └─[Security2CCMessageEncapsulation]
                                    │ sequence number: 218
                                    └─[SupervisionCCGet]
                                      │ session id:      30
                                      │ request updates: true
                                      └─[MultiChannelAssociationCCSet]
                                          group id:  1
                                          node ids:
                                          endpoints: 1:0
```
The command is acknowledged, but no response is sent and further communication fails for a few seconds.

Next steps: Check if the same happens when included via Mesh instead of LR.
